src/video-interface.js

```diff
@@ -76,8 +75,23 @@ export class VideoInterface {
    * AMP will not call this method if `controls` attribute is not set.
    */
   hideControls() {}
+
+  /**
+   * Automatically comes from {@link ./base-element.BaseElement}
+   *
+   * @return {!AmpElement}
+   */
+  get element() {}
```

Comment:
@jridgewell  yeah it's just interface def for type checking. It gets DC-ed. But good to know that getters are very slow!
@cramforce it does! Without this, compiler errored whenever I did `videoInterfaceInstance.element` saying element is not part of the interface definition.